README.md

```diff
@@ -1,127 +1,220 @@
-```c#
-var mirageSDKWrapper = MirageSDKWrapper.GetSDKInstance(ERC20ContractInformation.ProviderURL);
-```
+<!-- Installation -->
 
-To work with contracts you need to get a IContract instance from our SDKWrapper, for that call GetContract(...)
+##  🏗 Install SDK
 
-```c#
-string abi = "...";
-string contract_address = "0x...";
+1. Download `mirageSDK.unitypackage` from the latest release.
 
-IContract contract = mirageSDKWrapper.GetContract(contract_address, abi);
-```
+2. In your project, locate the 'Assets' folder. Move `mirageSDK.unitypackage` into this folder.
+
+3. 'Import all' to have access to all SDK capabilities.
+
+## 👀 What's in the SDK
+
+The SDK is designed to make it super easy to get started with Game development by enabling connection and interaction across different blockchains. 
+
+ * Contains a huge range of classes, plugins and example scripts for a variety of use cases. 
+
+* Nethereum libraries provide support for web requests using RPC over Http. 
+
+* Ankr RPC network infrastructure provides fast and easy connection to multiple chains. 
+
+
+## Getting started
+
+### 🧰 Prerequisites
+
+1. Smart Contracts must already be deployed on the blockchain. You have the Smart Contract addresses and ABI
+2. Web3.js or Ethers.js
+
+###  📌 Use Cases
 
-**IContract interface**
+This help content focuses on the following use cases. 
 
-* Task<string> CallMethod(string methodName, object[] arguments, string gas = null)*<br>
-Use for call methods of a contract deployed to blockchain.
-Returns string with transaction hash.
+1. Connecting to a Web3 Wallet (MetaMask) and Authenticating a user
+
+2. Performing Updates to NFTs by interacting with the blockchain and calling smart functions.
+
+
+<!-- 
+1. Examples Scripts, 
+*  The *Scripts* folder contains a wide range of useful scripts for initializing sessions, creating and managing Game NFTs and Scenes and authenticating sessions via a Web3 Wallet e.g. Metamask. 
+`ContractMessage` scripts for retrieving information for and managing NFTs  e.g. Game Characters on the blockchain.  ERC-721 and ERC-1155 
+`DTO` scripts for handling TransferObject events and accessing logs.
+`ERC-20`folder contains example script for minting new NFTs e.g. characters
+`ERC-721` folder contains example scripts for managing ownership of NFTs.
+`WearableNFTExample` folder contains example scripts for minting and managing NFT character apparel e.g. hats. 
+`ConnectionController` and `CustomSceneManager` ensure ongoing authentication via MetaMask and displaying game scenes. 
+to allow smart contract interfacing with the for building NFTs to the Blockchain.  scripts to ERC721 and ERC1155 scriptsScripts that can be used  -->
+
+## 👝 01 Connect Web3 Wallet
+
+Connecting to an Ethereum i.e. MetaMask wallet provides a link between a wallet address and a user's game account.
+
+1. Create an instance of a `Web3` class and call `Initialize` method after successful login in metamask
```

Comment:
Login is happening via WalletConnect instance in which we are using OpenDeepLink/OpenMobile wallet. Inside those methods we are using ConnectURL property to open a link.